src/components/App.jsx

```diff
@@ -20,17 +23,20 @@ const App = () => {
     setCurrentUser(user);
   }, []);
```

Comment:
this probably is async, need to use pattern:

`useEffect(() => {
    const callback() = async () => {
        const user = await getCurrentUser();
        setCurrentUser(user);
      }
      callback();
  }, [getCurrentUser]);`